### DISCUSSION TITLE
Examples of  "without loss of information" in 1.4.10?

### DISCUSSION THREAD

**Discussion by Ignacio-de-Loyola** [w3c/wcag#3265] [Q&A] [answered]
For example, when I zoomed in on the browser by 200%, the error message(Unable to select previous week.) was displayed on the screen, but when I zoomed in by 400%, the error message was not displayed on the screen I am currently viewing. (You can check the error message it when I scroll down)
[browser 200%]
![image](https://github.com/w3c/wcag/assets/80868256/64fbc342-e149-4c09-b603-11f63cf76a94)

[browser 400%]
![image](https://github.com/w3c/wcag/assets/80868256/2ba0887a-d3de-4648-a77a-ea9ddfc73824)

If the focus is in the date area and the focus is no longer available, an error message is displayed. If you move the focus down on the 400% zoomed screen, the error message is lost, so the keyboard user cannot see the error message below. Is this case "without loss of information"?

**Comment by patrickhlauke** [comment, marked as answer]
if you can still get to the error message (by scrolling), it is not "lost". the information is still there. it's natural that not everything you see at one glance in the viewport at one resolution may not be immediately there within the viewport at a smaller resolution. so no, not a failure.
